csrankings-j.csv

```diff
@@ -1256,6 +1256,7 @@ Jingtao Yao,University of Regina,http://www.cs.uregina.ca/People/Profiles/JingTa
 Jinguang Han,Queen's University Belfast,https://www.qub.ac.uk/schools/eeecs/Connect/Staff/BusinessCard/?name=j.han,euLvHOkAAAAJ
 Jingwei Li,UESTC,https://jingwei87.github.io,uSv67rIAAAAJ
 Jingwen Leng,Shanghai Jiao Tong University,https://www.cs.sjtu.edu.cn/~leng-jw/index.html,L1y8y2MAAAAJ
+Jingxian Wang,National University of Singapore,https://wangwireless.com,9j6NeHkAAAAJ

```

Comment:
Adjunct faculty members who are not full-time and on the campus of their university are not eligible for inclusion. Will close PR.

Reply:
Would you please open this PR again, as I'm full-time faculty. Thank you!